blocks/hero/hero.css

```diff
@@ -32,3 +32,91 @@ main .hero-container {
   width: 100%;
   height: 100%;
 }
+
+.hero.block {
```

Comment:
You can use nesting to namespace this better. For example:

```css
.hero {
    &.block {
        ... code
    }
}
```
Something like this.